pkg/operator/ceph/csi/csi.go

```diff
@@ -162,6 +162,11 @@ func (r *ReconcileCSI) setParams() error {
 		CSIParam.EnableCSIAddonsSideCar = true
 	}
 
+	CSIParam.EnableCrossNamespaceVolumeDataSource = false
+	if strings.EqualFold(k8sutil.GetOperatorSetting("CSI_ENABLE_CROSSNAMESPACEVOLUMEDATASOURCE", "false"), "true") {
```

Comment:
`CSI_ENABLE_CROSSNAMESPACEVOLUMEDATASOURCE` to `CSI_ENABLE_CROSS_NAMESPACE_VOLUME_DATA_SOURCE` for better readability?

Reply:
I renamed the variable